### ISSUE TITLE
Show information about the spectrometer's connection and scan status

### ISSUE THREAD

**Issue by alexdewar** [ImperialCollegeLondon/FINESSE#14]
Depends on #13.

This should be easy enough to do once the GUI is working and #13 has been achieved.

Aha! Link: https://imperial-college-ict.aha.io/requirements/RSE-122-13

**Comment by alexdewar** [comment]
Superseded by #34.
